nginx-config/nijobs.conf

```diff
@@ -0,0 +1,37 @@
+server {
+        # listen on port 80
+        listen 80;
+        # save logs here
+        access_log /var/log/nginx/access.log main;
+
+        # where the root here
+        root /usr/share/nginx/html;
+        # what file to server as index
+        index index.html index.htm;
+
+        location / {
+            # First attempt to serve request as file, then
+            # as directory, then fall back to redirecting to index.html
+            try_files $uri $uri/ /index.html;
+        }
+
+        # Media: images, icons, video, audio, HTC
+        location ~* \.(?:jpg|jpeg|gif|png|ico|cur|gz|svg|svgz|mp4|ogg|ogv|webm|htc)$ {
+          expires 1M;
+          access_log off;
+          add_header Cache-Control "public";
+        }
+
+        # Javascript and CSS files
+        location ~* \.(?:css|js)$ {
+            try_files $uri =404;
+            expires 1y;
```

Comment:
Is this desired? Will new versions of files hit cache and thus not update?

Reply:
I might be wrong, but in this setup, if there are "new files" there will be another container generated, therefore another nginx instance and no cache problems

Reply:
This cache is in the browsers. This is the header setting for that expiration, if I'm not mistaken

Reply:
It would make no sense for nginx to serve outdated versions of local files 🙃 the overhead would be bigger on creating that cache system than just serving them outright

Reply:
I've been researching, and this is actually useful. Every time we build and there is a change in the files, they change their name (they are appended with some kind of hash I believe). Therefore, if a build only changes some files that might only update some built ones and keep the others in cache, which is good

Reply:
Okay, good to go then!